crates/core/src/host/host_controller.rs

```diff
@@ -1215,7 +1227,9 @@ pub async fn extract_schema(program_bytes: Box<[u8]>, host_type: HostType) -> an
     let runtimes = HostRuntimes::new(None);
     let page_pool = PagePool::new(None);
     let core = SingleCoreExecutor::in_current_tokio_runtime();
-    let module_info = Host::try_init_in_memory_to_check(&runtimes, page_pool, database, program, core).await?;
+    let bsatn_rlb_pool = BsatnRowListBuilderPool::new();
```

Comment:
Maybe we want a variant of `extract_schema` that allows `HostController::migrate_plan` to pass the existing `PagePool`, `BsatnRowListBuildPool`, etc. ?